### PR TITLE
Add challenge DF into active rewards workflow

### LINKED ISSUE #656: Add "Challenge DF" into CI

### PR DESCRIPTION
**Pull request by trizin**
Fixes #656

Changes proposed in this PR:

- Add challenge DF into active rewards workflow

### CODE REVIEW

**Review by calina-c — CHANGES_REQUESTED** [review]

File: df_py/challenge/judge.py [hunk 1]
```diff
@@ -50,7 +50,7 @@ def _get_txs(deadline_dt) -> list:
 
     result = graphutil.submit_query(query_s, networkutil.network_to_chain_id("mumbai"))
     if "data" not in result:
-        print(f"_get_txs: An error occured, {result}")
+        raise Exception(f"_get_txs: An error occured, {result}")
```
Comment (calina-c) [inline df_py/challenge/judge.py:53]: can you please add a test as well?

**Review by calina-c — CHANGES_REQUESTED** [review]

**Review by calina-c — APPROVED** [review]

**Comment by calina-c** [comment]
@trizin I got this up to date and ready to merge (merged main, resolved all conversations etc.) but I will wait for you to merge it.

File: df_py/util/dftool_module.py [hunk 2]
```diff
@@ -316,6 +316,13 @@ def do_challenge_data():
             Example for Round 5: 2023-05-03_23:59
         """,
     )
+    parser.add_argument(
+        "--RETRIES",
+        default=1,
+        type=int,
+        help="# times to retry failed queries",
+        required=False,
+    )
```
Comment (idiom-bytes) [inline df_py/util/dftool_module.py:325]: nice 1

File: df_py/challenge/test/test_judge.py [hunk 3]
```diff
@@ -16,18 +18,20 @@ def test_get_txs():
...
+                "nftTransferHistories": [
+                    {
+                        "timestamp": six_days_ago.timestamp(),
+                        "nft": {"id": "0xnft1"},
+                        "oldOwner": {"id": "0xfrom1"},
+                    },
+                    {
+                        "timestamp": one_day_ago.timestamp(),
+                        "nft": {"id": "0xnft2"},
+                        "oldOwner": {"id": "0xfrom2"},
```
Comment (idiom-bytes) [inline df_py/challenge/test/test_judge.py:31]: I haven't looked at the rest of judge-related logic, and this is extremely minor/not the changelist. but, is `oldOwner` supposed to be `originalOwner`?

**Review by idiom-bytes — APPROVED** [review]
Looks good, thanks for the tests!

**Pull request merged**